R/init.R

```diff
@@ -46,6 +46,8 @@
 #'   See the vignette for an example. However, [ui_teal_with_splash()]
 #'   is then preferred to this function.
 #'
+#' @note If you use a module labelled `"Landing Popup"` `teal` will not create a tab for this module.
+#'
```

Comment:
not valid anymore

Reply:
yeah, just noticed as well and updated in here 11955bc32d7b9271ef3b0fd47166037be09cca7d